doc/howtos/OSTree.rst

```diff
@@ -0,0 +1,127 @@
+Handling System Updates Using OSTree
+####################################
+
+IoT Reference OS Kit ('refkit') has support for updating devices running
+one of the refkit profile images using an HTTP/HTTPs server and OSTree.
+Ostree is a library and suite of command line tools, largely inspired by
```

Comment:
Yes. I'd say when we switch from GPG signing to what Jussi have been cooking, dropping gpg1 and the rest would be a natural thing to do.